e2e_tests/tests/per_provider/normal_tests/hash.rs

```diff
@@ -11,10 +11,12 @@ const SHA_256: [u8; 32] = [
     0x0d, 0xc4, 0xbc, 0x13, 0xfd, 0x91, 0x74, 0x52, 0x92, 0x24, 0xc3, 0x8e, 0x0e, 0xe0, 0x75, 0xfa,
     0x9e, 0xd8, 0x0b, 0x78, 0x47, 0xe6, 0xae, 0xa7, 0x6a, 0xe9, 0x8c, 0xf9, 0xdd, 0xd9, 0x26, 0x69,
 ];
+#[cfg(not(feature = "cryptoauthlib-provider"))]
```

Comment:
Are those tests failing? Are yo utesting hashing with `hash_calib`?

Reply:
ATECC supports SHA256 only. And hash_calib.rs willl be removed.